packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox-group/gux-form-field-checkbox-group.scss

```diff
@@ -0,0 +1,26 @@
+@use '../../gux-form-field.scss';
+@use '../../functional-components/gux-form-field-fieldset-container/gux-form-field-fieldset-container.scss';
+@use '../../functional-components/gux-form-field-error/gux-form-field-error.scss';
+@use '../../functional-components/gux-form-field-legend-label/gux-form-field-legend-label.scss';
+@use '../../functional-components/gux-form-field-help/gux-form-field-help.scss';
+
+@include gux-form-field-fieldset-container.Style;
+@include gux-form-field-error.Style;
+@include gux-form-field-legend-label.Style;
+@include gux-form-field-help.Style;
+@include gux-form-field-error.Style;
+@include gux-form-field-help.Style;
+
+:host {
+  display: block;
+}
+
+:host([disabled]) {
+  .gux-form-field-legend-label {
+    opacity: var(--gse-ui-formControl-input-disabled-opacity);
+  }
+}
```

Comment:
This is not related to this PR but  when a checkbox is disabled you can still select/highlight the text.  This also happens if you have an error message or help message applied. I think by applying `user-select:none` should resolve it.